webpack.config.js

```diff
@@ -137,7 +137,8 @@ module.exports = env => {
         '/etc': aemDomain,
         '/designations': aemDomain
       },
-      port: 9000
+      port: 9000,
+      public: "localhost.cru.org:9000"
```

Comment:
This dev server is only for development. S3 serves the js everywhere else. This should be good.